.github/test-spec.yml

```diff
@@ -70,22 +71,6 @@
   - "samples/nrf9160/sms/**/*"
   - "subsys/net/lib/download_client/**/*"
 
-"CI-iot-lwm2m-client-test":
-  - "lib/nrf_modem_lib/**/*"
-  - "lib/at_monitor/**/*"
-  - "lib/at_cmd_parser/**/*"
-  - "lib/lte_link_control/**/*"
-  - "lib/modem_info/**/*"
-  - "lib/modem_key_mgmt/**/*"
-  - "lib/date_time/**/*"
-  - "subsys/net/lib/download_client/**/*"
-  - "subsys/net/lib/fota_download/**/*"
-  - "subsys/net/lib/lwm2m_client_utils/**/*"
-  - "subsys/dfu/dfu_target/**/*"
-  - "subsys/dfu/fmfu_fdev/**/*"
-  - "subsys/fw_info/**/*"
-  - "samples/nrf9160/lwm2m_client/**/*"
-
```

Comment:
No, just remove the LwM2M sample and `lwm2m_client_utils` from here. Rest, I belive, are relevant.

Reply:
This label doesn't exist in the downstream config anymore:

 https://projecttools.nordicsemi.no/bitbucket/projects/NCS-TEST/repos/test-fw-nrfconnect-nrf/browse/west.yml

It was first added by me and then changed later as we now use the `CI-iot-zephyr-lwm2m-test` label. 

So this is redundant and should be removed.